docs/04-configuration.md

```diff
@@ -20,6 +20,15 @@ Snowpack supports configuration files in multiple formats. Snowpack will look fo
 2. `snowpack.config.js`: A JS file exporting a config object (`module.exports = {...}`).
 3. `snowpack.config.json`: A JSON file containing config (`{...}`).
 
+#### Custom location
+
+If your configuration lives in a different location than any of the above, use the `--config` flag:
```

Comment:
can we move this documentation to the configuration table, and just reference the fact that this behavior exists here? (something like: "you can load configuration from a custom location via the --config flag").

Reply:
Yeah I was thinking about that. The tricky part is this is one of the only cli-only options. It doesn’t live in `installOptions` because… it’s the path to config!

<img width="976" alt="Screen Shot 2020-03-03 at 18 54 06" src="https://user-images.githubusercontent.com/1369770/75836845-62065280-5d80-11ea-904f-595afb24af7e.png">

Should we add a note or something saying “CLI flag only?” is that weird being under `installOptions*` heading?

Reply:
... you know what... I went ahead and added these cli-only flags to the table in #228 without realizing that the table header was "Install Options (`installOptions.*`)".

We have enough now that it makes sense to split out. how about we create a new, identical table up in the 'CLI flags" section above?

Reply:
LMK what you think. I copied the table, and now in the “Config File” section I deleted the 3 options that had CLI flags but not config file parallels.

Reply:
I went ahead and merged it but as always feel free to rewrite that docs section any way you like.

Reply:
+1, docs should never be blocking! This looks good, will find a way to simplify on master so that we don't need to duplicate the table twice.